backend/src/main/java/com/htecgroup/skynest/model/entity/FileMetadataEntity.java

```diff
@@ -37,6 +37,10 @@ public boolean isDeleted() {
         || (parentFolder == null ? bucket.isDeleted() : parentFolder.isDeleted());
   }
 
+  public boolean someParentIsDeleted() {
```

Comment:
you could call this in `isDeleted()` above

Reply:
we could do the same for folders

Reply:
Added this